.dockerignore

```
@@ -0,0 +1,4 @@
+.git
+*~
+cache
+gcloud.json
```

```suggestion
gcloud.json
**/*.pyc
**/__pycache__/
```

We likely want to ignore any local python bytecode.